test/e2e/runner/start.go

```diff
@@ -29,9 +32,7 @@ func Start(testnet *e2e.Testnet) error {
 	sort.SliceStable(nodeQueue, func(i, j int) bool {
```

Comment:
This is pre-existing, but I noticed it in reading: It's odd that we bothered to "copy" this out of `testnet.Nodes`, since the sort will have permuted the underlying slice in-place. It's probably harmless, but it sort of looks like the intent was to sort a copy, which we are not doing (since slices are references).

Reply:
_yup_